sympy/geometry/tests/test_line.py

```diff
@@ -54,13 +54,13 @@ def test_line_geom():
     raises(ValueError, lambda: Line(Point(0, 0), Point(0, 0)))
 
     # Basic stuff
-    assert Line((1, 1), slope=1) == Line((1, 1), (2, 2))
```

Comment:
Thank you for your comment. You are probably right, that adding `equals()` calls separately would be better. 
I will revise my changes and I will leave these `equals()` calls, which really improve coverage. For example this usage of `equals` seems quite reasonable for me (of course two asserts are valuable), because it checks mathematical identity, which in this case is not implied by `==` equality. The objects are different but they can differ because of different reasons:
```python 
-    assert l1 != l2
+    assert l1.equals(l2)
```

BTW: I'm going to take a look on non-test code now, but I'm thinking also about granulating this file - I mean many different test cases are hidden under `test_line_geom` name and they could be split into many smaller functions, which would improve a readability (e.g. some variables are assigned many times and it's hard to follow the code). What do you think about it?